templates/case-study/esa-case-study.html

```diff
@@ -0,0 +1,215 @@
+{% extends "case-study/base_case-study.html" %}
+{% from "macros/_macros-hero.jinja" import hero %}
+{% from "macros/_macros-text-list.jinja" import text_list %}
+{% from "macros/_macros-highlights.jinja" import highlights %}
+{% from "macros/_macros-text.jinja" import text %}
+{% from "macros/_macros-quote.jinja" import quote_wrapper %}
+{% from "macros/_macros-image.jinja" import image_wrapper %}
+
+{% block title %}Canonical deploys infrastructure solutions and managed IT services for critical space mission operations{% endblock %}
+{% block meta_description %}Example meta_description{% endblock %}
```

Comment:
Please put a real text here as it shows on the card in the overview page

Reply:
I removed the meta_description since the [original case study](https://pages.ubuntu.com/rs/066-EOV-335/images/European%20Space%20Agency%20Case%20Study%20April%202024%20.pdf?version=0&_ga=2.107877442.1558731246.1724143218-905304663.1722502159&_gl=1*1r8t18f*_gcl_au*OTk2MTIyNDYwLjE3MjMxOTI5NjU.) does not have one